src/test/scala/com/fulcrumgenomics/alignment/NeedlemanWunschAlignerTest.scala

```diff
@@ -24,29 +24,37 @@
 
 package com.fulcrumgenomics.alignment
 
-import com.fulcrumgenomics.alignment.Mode.Glocal
+import com.fulcrumgenomics.alignment.Mode.{Glocal, Local}
 import com.fulcrumgenomics.testing.UnitSpec
 
 class NeedlemanWunschAlignerTest extends UnitSpec {
   /** Upper-cases and remove display-related characters from a string. */
   def s(str: String): String = str.filterNot(ch => ch == '-').toUpperCase
 
-  /** Asserts a couple of things about an alignment that should be constant for global alignments. */
+  /** Asserts a couple of things about alignment that should be constant for global alignments. */
```

Comment:
I think this read better before.  Alignment needs an article before it, either `an` or `the`.